File: Server/src/main/java/com/complex/server/responses/NotificationResponse.java
package com.complex.server.responses;

import com.complex.server.requests.Date;
import com.complex.server.requests.DateTime;

public record NotificationResponse(
        Long id,
        Integer eventId,
        String title,
        String description,
        Date sent,
        DateTime start,
        DateTime end,
        UserResp user
) {}

File: ComplexProject/src/app/services/responses/NotificationResponse.ts
import {Data} from "./Data";
import {DateTime} from "./DateTime";
import {UserResponse} from "./UserResponse";

export class NotificationResponse{
  id: number
  eventId: number
  title: String
  description: String
  sent: Data
  start: DateTime
  end: DateTime
  user: UserResponse

  constructor(id: number, eventId: number, title: String, description: String, sent: Data, start: DateTime, end: DateTime, user: UserResponse) {
    this.id = id;
    this.eventId = eventId;
    this.title = title;
    this.description = description;
    this.sent = sent;
    this.start = start;
    this.end = end;
    this.user = user;
  }

}
